frontend/pages/admin/TeamManagementPage/components/DeleteTeamModal/DeleteTeamModal.tsx

```diff
@@ -31,7 +31,7 @@ const DeleteTeamModal = ({
           <span className={`${baseClass}__name`}>{name}</span> from Fleet.
         </p>
         <p>
-          Members of this team who are not members of other teams will lose
+          Users of this team who are not assigned to other teams will lose
```

Comment:
One more small change to keep the language consistent:
```suggestion
          Users on this team who are not assigned to other teams will lose
```